packages/mock-account-provider/app/routes/webhooks.ts

```diff
@@ -107,8 +109,14 @@ export async function handleOutgoingPaymentCreated(wh: WebHook) {
   return json(undefined, { status: 200 })
 }
 
-export async function handleIncomingPaymentCompleted(wh: WebHook) {
-  assert.equal(wh.type, EventType.IncomingPaymentCompleted)
+export async function handleIncomingPaymentCompletedExpired(wh: WebHook) {
+  if (
+    wh.type !== EventType.IncomingPaymentCompleted &&
+    wh.type !== EventType.IncomingPaymentExpired
+  ) {
+    assert.fail('invalid event type')
```

Comment:
Not sure if it's standard, but I know there are some `assert`s sprinkled throughout rafik (in tests and otherwise)i.
In this case, maybe you could do something like:
```suggestion
    throw json('Internal error', { status: 500 })
```
At some point, it be nice to add OpenAPI validation to the mock account provider which could allow us to pass specific typed events to these handlers and remove these checks.